swagger.yaml

```diff
@@ -1,11 +1,235 @@
+    /store/brands/{Id}/products:
+        get:
+            tags:
+                - 'Store'
+            summary: 'Returns products by brand'
+            description: 'Returns all products associated with a certain brand ID'
+            parameters:
+                - name: Id
+                  in: path
+                  description: ID of sunglasses brand to be fetched
+                  required: true
+                  type: string
+            responses:
+                200:
+                    description: Successful Request
+                    schema:
+                        type: array
+                        items:
+                            $ref: '#/definitions/Product'
+                404:
+                    description: Item not found
+    /store/products:
+        get:
+            tags:
+                - 'Store'
+            summary: Returns all items in the store
+            description: Returns a list of all items in the store
+            parameters:
+                - name: query
+                  in: query
+                  required: false
+                  type: string
+            responses:
+                200:
+                    description: Successful Request
+                    schema:
+                        type: array
+                        items:
+                            $ref: '#/definitions/Product'
+                404:
+                    description: Inventory not found
+    /login:
+        post:
+            tags:
+                - 'User'
+            summary: User Login
+            description: Logs in the user and returns access token
+            parameters:
+                - in: body
+                  name: credentials
+                  required: true
+                  schema:
+                      $ref: '#/definitions/Credentials'
+            responses:
+                200:
+                    description: Successful Login
+                    schema:
+                        $ref: '#/definitions/LoginResponse'
+                400:
+                    description: Invalid username or password
+                401:
+                    description: Unauthorized - Invalid Credentials Format
+    /me/cart:
+        get:
+            tags:
+                - 'User'
+            summary: Get Users shopping cart
+            description: Returns the current users shopping cart
+            parameters:
+                - name: accessToken
+                  in: query
+                  required: true
+                  type: string
+            responses:
+                200:
+                    description: Successful Request
+                    schema:
+                        type: array
+                        items:
+                            $ref: '#/definitions/CartItem'
+                401:
+                    description: Unauthorized - User not authenticated. Please sign in
+        post:
+            tags:
+                - 'User'
```

Comment:
I would have separated User actions like login with the Cart actions. In the future when your application is bigger, it wont be sustainable.